implementation.js

```diff
@@ -138,6 +152,9 @@ module.exports = function from(arrayLike) {
 	}
 	var items = ES.ToObject(arrayLike);
 
+	// variables for rebuilding array from iterator or string.
+	items = parseES6(items);
+
 	var mapFn, T;
 	if (typeof arguments[1] !== 'undefined') {
 		mapFn = arguments[1];
```

Comment:
I see, I'll place new lines before `var len = ES.ToLength(items.length);` to keep the order of exceptions.